src/Agent.Listener/Configuration/Validators.cs

```diff
@@ -89,5 +91,17 @@ public static bool NTAccountValidator(string arg)
 
             return true;
         }
+
+        public static bool MachineGroupTagsValidator(string tags)
+        {
+            if (string.IsNullOrEmpty(tags))
+            {
+                return false;
+            }
+
+            var tagList = tags.Split(',');
+
+            return tagList.All(tag => tag.Trim().Length <= 256);
```

Comment:
do we really need the length check here? as Krishna said, when you fail, you can't tell user what's wrong